src/main/java/uk/nhs/hee/tis/trainee/credentials/service/JwtService.java

```diff
@@ -116,4 +118,19 @@ public Claims getClaims(String token, boolean verifySignature) {
       return parser.parseClaimsJws(token).getBody();
     }
   }
+
+  /**
+   * Get the mapped token body, the header or signature will not be verified.
+   * TODO: investigate why Authorization token is not valid for getClaims()
```

Comment:
`getClaims()` has an overload now which handles additional scenarios, so this method should be able to be removed.